### ISSUE TITLE
Crash Hyprland possible related with Xwayland(Chromium)

### ISSUE THREAD

**Issue by hikkidev** [hyprwm/Hyprland#1126]
- Hyprland, built from branch main at commit [3dd06b674a252c51937d20ffecfc0274ce63cfff](https://github.com/hyprwm/Hyprland/commit/3dd06b674a252c51937d20ffecfc0274ce63cfff) dirty (clean up blur code and fix minor issues). flags: (if any)
- Chromium 107.0.5304.110-1
- Xorg-xwayland 22.1.5-1 (xorg)

## Steps to reproduce
1) Launch chromium
2) Observe Hyprland close

## Expected outcome
- Hyprland is stable

## The Hyprland log
https://hastebin.com/edogijiqas.sql

## Coredump / Coredump analysis (with a stacktrace)
```
Tue 2022-11-29 14:06:59 +07   1877 1000 1000 SIGABRT present  /usr/bin/Hyprland                                                                          
Tue 2022-11-29 14:06:59 +07   2709 1000 1000 SIGABRT present  /usr/bin/Xwayland
```
Hyprland: https://hastebin.com/icigecahah.yaml
XWayland: https://hastebin.com/qenasovule.yaml

## Config
https://hastebin.com/zixawofihi.bash

**Comment by vaxerski** [comment]
according to your comment on discord, this is a regression.

git bisect then and provide the faulty commit.

**Comment by hikkidev** [comment]
I'm not a member of the Hyprland discord server

**Comment by vaxerski** [comment]
Oh. Is this a regression?

**Comment by hikkidev** [comment]
No. I used the binary version [hyprland-bin](https://aur.archlinux.org/packages/hyprland-bin), but then @DashieTM asked me (https://github.com/hyprwm/Hyprland/issues/1074)  to test a new rule. So, I built a version from git.

**Comment by vaxerski** [comment]
"and found this issue" so it didn't exist on -bin?

**Comment by hikkidev** [comment]
yeah

**Comment by vaxerski** [comment]
then that *is* a regression. Please do a git bisect and provide the first faulty commit.

**Comment by hikkidev** [comment]
I understand what regression testing is. What I meant was that I was not purposefully testing of all Hyprland, but found this issue by accident. And I don't have the resources for testing at the moment. As soon as I'm free, I'll write back.

**Comment by vaxerski** [comment]
fixed in 55776df685340ae798aace1c74381d3f6d350313
